src/libostree/libostree.sym

```diff
@@ -89,6 +89,7 @@ global:
         ostree_deployment_set_index;
         ostree_deployment_set_origin;
         ostree_diff_dirs;
+        ostree_diff_dirs_with_options;
```

Comment:
This will need to be in the `LIBOSTREE_2017.2` section down below - the section here is for symbols already released.